docs/managing-dragonfly/operator/dragonfly-configuration.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+---
+
+# Dragonfly CRD
+
+Dragonfly Operator uses a dedicated Dragonfly CRD to create and manage dragonfly
```

Comment:
Small nit, caps the `d` in dragonfly. That is `Dragonfly` instead of `dragonfly`